core/lib/resources/resource.js

```diff
@@ -134,7 +133,7 @@ Resource.prototype.handleMessage = function (socket, message) {
 Resource.prototype.destroy = function () {}
 
 Resource.setBackend = function (backend) {
-  Persistence = backend
+  // Persistence = backend
```

Comment:
it looks like it's part of an abstract interface used for all resources (MessageList, etc). I'll comment it as a no-op, but if we want to change that interface it is out of scope for this pr